inference-engine/ie_bridges/java/tests/InferRequestTests.java

```diff
@@ -12,15 +16,16 @@
     InferRequest inferRequest;
     boolean completionCallback;
 
-    @Override
-    protected void setUp() {
+    @Before
+    public void setUp() {
         core = new IECore();
         net = core.ReadNetwork(modelXml);
         executableNetwork = core.LoadNetwork(net, "CPU");
```

Comment:
I think, we shouldn't create ```core```, ```net``` and ```executableNetwork``` before each test, we can make it once.

Reply:
Do you have a plan to extend these tests for other plugins?

Reply:
Added a parameter